.github/workflows/aws_setup.yml

```diff
@@ -0,0 +1,25 @@
+name: aws-resource-setup
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  apply:
+    runs-on: ubuntu-latest
+    name: Apply approved plan
+    if: github.event.pull_request.head.repo.full_name == github.repository
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Terraform apply AWS resources
+        uses: dflook/terraform-apply@v1.4.1
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+        with:
+          path: aws-resources.tf
```

Comment:
@HubertStefanski why don't you use another step in current `a-team` flow instead of creating a new workflow?

Should this workflow run every time a package or something in this repo or just when the `saw-resources.tf` file changes?

Reply:
True, you're right, I should probably add this to a stage before we kick off the build rather than have it as a separate run, thanks for the feedback :+1:

Reply:
> Should this workflow run every time a package or something in this repo or just when the `saw-resources.tf` file changes?
every time. it should help us avoid a configuration drift.